plugins/workspace-minimap/src/minimap.ts

```diff
@@ -97,12 +100,16 @@ export class Minimap {
       this,
       this.onClickDown,
     );
-    this.onMouseUpWrapper = Blockly.browserEvents.bind(
-      primaryInjectParentDiv,
-      'mouseup',
-      this,
-      this.onClickUp,
-    );
+    if(primaryInjectParentDiv === null) {
+      throw new Error("primaryInjectParentDiv is null");
```

Comment:
```suggestion
    if (!primaryInjectionDiv) {
      throw new Error('The workspace must be injected into the page before the minimap can be initalized');
```
Can you also move this to the top of the `init` method? The `this.onMouseUpWrapper` assignment can be extracted from the `else` and left where it is.